INSTALLATION.MD

```diff
@@ -0,0 +1,181 @@
+# Linux
```

Comment:
This lacks a good header, an overview to the available options and maybe an introduction.